### PR TITLE
Added : Implemented code for Azure Elastic database support.

### PR DESCRIPTION
**Pull request by daisukenishino**
This is a research to correspond to the following enhancement.

- [To support data providers of "Elastic Database of Azure SQL Database". #144](https://github.com/OpenTouryoProject/OpenTouryo/issues/144)

It is planed to develop a new "dam" corresponding to Azure Elastic database.
This project will be used in this feasibility study.

### CODE REVIEW

File: Azure/ElasticDatabase/ElasticDB_Sample/ElasticDB_Sample.sln [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Express 2012 for Web
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ElasticScaleStarterKit", "ElasticScaleStarterKit\ElasticScaleStarterKit.csproj", "{576E2FB9-A842-457C-BA5B-5E53A2FA11A9}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{33AF7A53-A66B-4667-9C05-EE040AFA812B}"
+	ProjectSection(SolutionItems) = preProject
+		..\..\..\..\USERS\INSPANDA03\APPDATA\LOCAL\MICROSOFT\VWDEXPRESS\11.0\EXTENSIONS\141M0CDP.Z00\description.html = ..\..\..\..\USERS\INSPANDA03\APPDATA\LOCAL\MICROSOFT\VWDEXPRESS\11.0\EXTENSIONS\141M0CDP.Z00\description.html
```
Comment (yusukemaegawa) [inline Azure/ElasticDatabase/ElasticDB_Sample/ElasticDB_Sample.sln]: @daisukenishino 

This file could not be found.
So, I could not open this file.

Is this file necessary?

**Comment by daisukenishino** [comment]
@yusukemaegawa 

I have deleted "description.html" file in "Solution Items" folder.
see : https://github.com/SymphonyTeleca/SampleProgram/commit/71a851907969cefba6a8cc6889343632be37cb04
